packages/calcite-components/src/components/list/list.tsx

```diff
@@ -533,7 +538,14 @@ export class List implements InteractiveComponent, LoadableComponent {
     const filteredItems = this.enabledListItems.filter((listItem) => this.isNavigable(listItem));
     const currentIndex = filteredItems.findIndex((listItem) => listItem.active);
 
-    if (key === "ArrowDown") {
+    if (key === "ArrowDown" && event.target === this.filterEl) {
```

Comment:
Could you move this into the `ArrowDown` check below and only update `nextIndex` to DRY it up?